tests/storage/guestfs.go

```diff
@@ -69,11 +69,10 @@ var _ = SIGDescribe("[rfe_id:6364][[Serial]Guestfs", func() {
 		return f
 	}
 
-	runGuestfsOnPVC := func(pvcClaim string) {
+	runGuestfsOnPVC := func(pvcClaim string, options []string) {
```

Comment:
You can use `options ...string` and don't need to write `[]string{}` in the tests